ci-operator/step-registry/cucushift/ota/postupgrade/cucushift-ota-postupgrade-commands.sh

```diff
@@ -224,6 +224,30 @@ function post-OCP-69948(){
     return 0
 }
 
+function post-OCP-56083(){
+    echo "Post Test Start: OCP-56083"
+    echo "Upgrade cluster when channel is unset"
+    local tmp_log expected_msg result
+    tmp_log=$(mktemp)
+    expected_msg='\"acceptedRisks\": \"Precondition \\\"ClusterVersionRecommendedUpdate\\\" failed because of \\\"NoChannel\\\": Configured channel is unset, so the recommended status of updating from'
+    result=$(oc get clusterversion -ojson | jq -r '.items[].status.conditions[]|select(.type == "ReleaseAccepted").status')
+    if  [[ "${result}" == "True" ]]; then
+        oc get clusterversion -ojson | jq -r '.items[].status.history[0]'  2>&1 | tee ${tmp_log} || true
```

Comment:
```suggestion
        oc get clusterversion -ojson | jq -r '.items[].status.history[0].acceptedRisks' 
```
I think we can directly get the value of `acceptedRisks`, we can directly compare `expected_msg` with the output, not have to use `grep` way.